app/src/main/java/ca/marshallasch/veil/ListAdapter.java

```diff
@@ -0,0 +1,63 @@
+package ca.marshallasch.veil;
```

Comment:
Please rename this class to be more descriptive with what it is for,

Reply:
Renamed to ListCell

Reply:
Im updating this to `PostListAdapter` Because the class is still an adapter not a cell, but we may have multiple adapters later on.